devtools/maven/src/main/java/io/quarkus/maven/ListExtensionsMojo.java

```diff
@@ -23,19 +23,25 @@
     protected MavenProject project;
 
     /**
-     * list all extensions or just the installable.
+     * List all extensions or just the installable.
      */
     @Parameter(property = "quarkus.extension.all", alias = "quarkus.extension.all", defaultValue = "true")
     protected boolean all;
 
     /**
-     * display in simplified format.
+     * Display in simplified format.
      */
     @Parameter(property = "quarkus.extension.format", alias = "quarkus.extension.format", defaultValue = "full")
     protected String format;
 
+    /**
+     * Search filter on extension list. The format is based on Java Pattern.
+     */
+    @Parameter(property = "search", alias = "quarkus.extension.search")
+    protected String search;
```

Comment:
How about we call it `pattern`?

Reply:
IMO pattern is less user friendly, because pattern semantic not contains the filter notion.
But I am not english native, so I can be wrong.

Reply:
The value of this parameter is a pattern, isn't it? I'm simply proposing to call it what it is. To me "list... pattern=..." reads better than "list... search=..." (two verbs). That'd just be my preference. Let's see what others think.
Thanks for another contribution!

Reply:
I'd suggest changing it, and the parameter names in `ListExtensions` to `searchPattern`. To me it's then clear that it's a pattern for searching for an extension